spec/helpers.rb

```diff
@@ -125,24 +125,10 @@ module PlatformHelpers
   FEDORASYSTEMD    = (14..39).to_a.collect! { |i| "fedora-#{i}" }
 
   SYSTEMDPLATFORMS = %w[amazon-2023
-                        el-7
-                        centos-7
-                        redhat-7
-                        oracle-7
-                        scientific-7
-                        eos-7
-                        el-8
-                        centos-8
-                        redhat-8
-                        oracle-8].concat(FEDORASYSTEMD)
-
-  FEDORASYSTEMV    = (1..13).to_a.collect! { |i| "fedora-#{i}" }
-
-  SYSTEMVPLATFORMS = ['el-',
-                      'centos',
-                      'fedora',
-                      'redhat',
-                      'oracle',
-                      'scientific',
-                      'eos',].concat(FEDORASYSTEMV)
+                        el-
+                        centos
+                        redhat
+                        oracle
+                        scientific
+                        eos].concat(FEDORASYSTEMD).concat(DEBIANPLATFORMS)
```

Comment:
If you're going this route, just get rid of `FEDORASYSTEMD` too. Fedora < 14 is VERY old by now. You may also get rid of `DEBIANPLATFORMS` too. In that case, the distinction really becomes Linux vs non-Linux.

Reply:
Debian is used to test e.g. apt-get commands too

Reply:
https://github.com/voxpupuli/beaker/pull/1873